stumpy/snippets.py

```diff
@@ -187,6 +209,16 @@ def snippets(
         The p-norm to apply for computing the Minkowski distance. This parameter is
         ignored when `normalize == True`.
 
+    T_subseq_isconstant : numpy.ndarray or function, default None
```

Comment:
It feels like we should be calling it `mpdist_T_subseq_isconstant`?

Reply:
I know it's a LOOOOONG name

Reply:
It is long but it clearly separates it from other cases where window size is `m`. Also, we are just passing it to another function and it does not appear that much in the code. So, I think it is good :) I will revise the code.